dbms/src/TestUtils/FunctionTestUtils.h

```diff
@@ -816,6 +823,21 @@ class FunctionTest : public ::testing::Test
         return executeFunction(func_name, argument_column_numbers, vec);
     }
 
+    ColumnWithTypeAndName executeFunctionWithMetaData(
+        const String & func_name,
+        const ColumnsWithTypeAndName & columns,
+        const TiDB::TiDBCollatorPtr & collator = nullptr,
+        const FuncMetaData & meta = FuncMetaData(),
+        bool raw_function_test = false);
```

Comment:
I think here is no need to add `raw_function_test ` for `executeFunctionWithMetaData ` because function with meta can't execute in raw_function_test mode, and also I think since `executeFunctionWithMetaData ` is a new function that should only be used for function that has function meta, there is no need to set default value of `meta`

Reply:
> I think here is no need to add `raw_function_test ` for `executeFunctionWithMetaData ` because function with meta can't execute in raw_function_test mode, and also I think since `executeFunctionWithMetaData ` is a new function that should only be used for function that has function meta, there is no need to set default value of `meta`

I will change it